_posts/2020-05-18-mirror-kafka-eventhub.md

```diff
@@ -0,0 +1,219 @@
+---
+layout: post
+title:  "Mirroring from Apache Kafka to Azure Event Hub"
+date: 2020-05-18
+author: paolo_patierno
+---
+
+Let's imagine to have a software architecture based on Apache Kafka running on an on-premise Kubernetes or OpenShift installation and managed by the Strimzi operators.
```

Comment:
I wonder if we really need to mention OpenShift here when the whole demo is based on Kubernetes.

Reply:
Well, I am saying "or OpenShift" as another Kubernetes distribution. During the demo, I had to choose one of course.

Reply:
Which is a bit weird since it doesn't mention the 100 other Kubernetes distributions but just one specific although it doesn't have anything really related to the blogpost..